src/main/java/com/scalar/database/api/DistributedTransactionManager.java

```diff
@@ -4,10 +4,42 @@
 
   void with(String namespace, String tableName);
 
+  /**
+   * Starts a new transaction.
+   *
+   * @return {@link DistributedTransaction}
+   */
   DistributedTransaction start();
 
+  /**
+   * Starts a new transaction with the specified transaction ID. It is a users' responsibility to
```

Comment:
Regarding "a users' responsibility". Probably it should be "It is the user's responsibility..." or "It is users' responsibility ...". Similarly on lines 33-34.

Reply:
Cool. Thanks ! Will change it.

Reply:
Fixed